src/main/resources/application.properties

```diff
@@ -4,8 +4,3 @@ spring.jpa.hibernate.ddl-auto=update
 spring.jpa.properties.hibernate.format_sql=true
 logging.level.org.hibernate.SQL=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-security.authentication.jwt.expirationtime=86400000
-security.authentication.jwt.secret=switchboardsecret
-security.authentication.superuser.username=admin
-security.authentication.superuser.password=admin
-security.authentication.url=/auth/token
```

Comment:
You don't want to keep the authentication.url to be configurable?

Reply:
Configuration is done in SecurityProperties.java, no need to add them here.

Reply:
Also "/auth/token" was never being used, so, not too sure what it was all about in the first place? The SecurityProperties was always using "/login" as the authentication url

Reply:
Gotcha, I would change the value of AUTHORIZATION_URL in SecurityProperties.java to `/auth/token` otherwise it'll break the FE.